static/app/components/discover/performanceCardTable.tsx

```diff
@@ -0,0 +1,667 @@
+                  project: `${releaseEventView.project}`,
+                  field: DiscoverField.CUMULATIVE_LAYOUT_SHIFT,
+                  statsPeriod: `${releaseEventView.statsPeriod}`,
+                },
+              }}
+            >
+              {performanceCardLabels.cumulativeLayoutShift}
+            </Link>
+          </Field>
+          <SubTitle>{performanceCardLabels.spanOperations}</SubTitle>
+          <Field>
+            <Link
+              to={{
+                pathname: discoverPath,
+                query: {
+                  query: `event.type:transaction ${releaseEventView.query}`,
+                  project: `${releaseEventView.project}`,
+                  field: DiscoverField.SPANS_HTTP,
+                  statsPeriod: `${releaseEventView.statsPeriod}`,
+                },
+              }}
+            >
+              {performanceCardLabels.http}
+            </Link>
+          </Field>
+          <Field>
+            <Link
+              to={{
+                pathname: discoverPath,
+                query: {
+                  query: `event.type:transaction ${releaseEventView.query}`,
+                  project: `${releaseEventView.project}`,
+                  field: DiscoverField.SPANS_DB,
+                  statsPeriod: `${releaseEventView.statsPeriod}`,
+                },
+              }}
+            >
+              {performanceCardLabels.db}
+            </Link>
+          </Field>
+          <Field>
+            <Link
+              to={{
+                pathname: discoverPath,
+                query: {
+                  query: `event.type:transaction ${releaseEventView.query}`,
+                  project: `${releaseEventView.project}`,
+                  field: DiscoverField.SPANS_BROWSER,
+                  statsPeriod: `${releaseEventView.statsPeriod}`,
+                },
+              }}
+            >
+              {performanceCardLabels.browser}
+            </Link>
+          </Field>
+          <Field>
+            <Link
+              to={{
+                pathname: discoverPath,
+                query: {
+                  query: `event.type:transaction ${releaseEventView.query}`,
+                  project: `${releaseEventView.project}`,
+                  field: DiscoverField.SPANS_RESOURCE,
+                  statsPeriod: `${releaseEventView.statsPeriod}`,
+                },
+              }}
+            >
+              {performanceCardLabels.resource}
+            </Link>
+          </Field>
+        </StyledPanelItem>
+        <StyledPanelItem>{this.renderResults()[0]}</StyledPanelItem>
+        <StyledPanelItem>{this.renderResults()[1]}</StyledPanelItem>
+        <StyledPanelItem>{this.renderChangeResults()}</StyledPanelItem>
+      </Fragment>
+    );
+  }
+
+  renderBackendPerformance() {
+    const {organization, releaseEventView} = this.props;
+    const discoverPath = `/organizations/${organization.slug}/discover/results/`;
```

Comment:
Who has access to this endpoint? Is it only customers with Discover, or also Performance?

Reply:
I think customers with at least `discover-basic` - I found a feature component I might be able to use for this